component.html

```diff
@@ -134,6 +135,146 @@ <h3 class="avatar-heading">Alert</h3>
+        Badges are used to highlight an item's status for quick recognition.Badges scale to match the size of the immediate parent element by using relative font sizing and rem units.
+      </p>
+      <h3 class="avatar-heading">Badge with Icons</h3>
+      <p class="component-navbar-item">
+        To show cart or notification count, you can use number badge. Check code below to copy the html part as is.
+      </p>
+      <div class="badge-items flex">
+        <button type="button" class="badge-button flex center badge-xl">
+          <i class="fas fa-bell fa-4x"></i>
+          <span class="icon-button-badge flex center icon-xl">2</span>
+        </button>
+        <button type="button" class="badge-button flex center badge-lg">
+          <i class="fas fa-envelope fa-3x"></i>
+          <span class="icon-button-badge flex center icon-l">2</span>
+        </button>
+        <button type="button" class="badge-button flex center badge-md">
+          <i class="fab fa-whatsapp fa-2x"></i>
+          <span class="icon-button-badge flex center icon-md">2</span>
+        </button>
+      </div>
+
+      <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28248%2C192%2C200%2C0.94%29&t=base16-light&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=39px&ph=50px&ln=false&fl=1&fm=Droid+Sans+Mono&fs=13.5px&lh=130%25&si=false&es=2x&wm=false&code=%250A%253Cbutton%2520type%253D%2522button%2522%2520class%253D%2522badge-button%2520badge-xl%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-bell%2520fa-4x%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%253Cspan%2520class%253D%2522icon-button-badge%2520icon-xl%2522%253E2%253C%252Fspan%253E%250A%253C%252Fbutton%253E%250A%253Cbutton%2520type%253D%2522button%2522%2520class%253D%2522badge-button%2520badge-lg%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-envelope%2520fa-3x%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%253Cspan%2520class%253D%2522icon-button-badge%2520icon-l%2522%253E2%253C%252Fspan%253E%250A%2520%253C%252Fbutton%253E%250A%2520%253Cbutton%2520type%253D%2522button%2522%2520class%253D%2522badge-button%2520badge-md%2522%253E%250A%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fab%2520fa-whatsapp%2520fa-2x%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522icon-button-badge%2520icon-md%2522%253E2%253C%252Fspan%253E%250A%2520%253C%252Fbutton%253E%250A%250A"
+  style="width: 100%; height: 398px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+      
+      <h3 class="avatar-heading">Badge with avatar</h3>
+      <p class="component-navbar-item">
+        Badges can be integrated with Avatars by showing colors to show the online status of user. 
+      </p>
+
+      <div class="badge-items flex">
+        <div class="badge-avatar flex  badge-xl">
+          <img src="./Asset/kitty.png" alt="" class="kitty avatar-xl" /><span
```

Comment:
It would be good if you give value to alt

Reply:
Okay